translate/rus.json

```diff
@@ -211,9 +211,9 @@
 "line":"линия",
 "linear":"линейный",
 "links":"ссылки",
-"live engine 1":"живой движок 1",
-"live engine 2":"живой движок 2",
-"live log":"живой журнал",
+"kibitz engine 1":"движок-советчик 1",
+"kibitz engine 2":"движок-советчик 2",
+"live log":"log-журнал",
```

Comment:
Lines 214-215: Current Stoof and KD aren't called kibitzers for nothing, right? These adjustments are made just for that little issue. But again, translations for other languages have to be made by you since I don't know them.
Line 216: More direct and short translation for Rus Loc. Now it clearly says that it is a 'log-journal' and nothing else.